www/index.html

```diff
@@ -62,6 +62,7 @@ <h1>Hi I&rsquo;m Markus</h1>
         <dd>
           <a href="https://dnszensur.ch/">DNSzensur</a>,
           <a href="https://web.archive.org/web/20171022202658/https://ueberwacht.ch/">ueberwacht.ch</a>
+	  <a href="https://firefoxvschrome.com/">firefoxvschrome.com</a>
```

Comment:
Can you please use spaces for indentations?

Reply:
fixed